pages/[id].jsx

```diff
@@ -1,26 +1,29 @@
 import Head from 'next/head';
 import SkipToContent from '../components/SkipToContent';
 import apiService from '../services/apiService';
+import getFullName from '../util/getFullName';
 
 export default function DetailPage({ fetchedTeamMember }) {
+  const { firstName, lastName } = fetchedTeamMember;
+  const fullName = getFullName(firstName, lastName);
```

Comment:
I named the function `getFullName` so our linter wouldn't get angry about naming the return value variable here `fullName`.